include/aquamarine/output/Output.hpp

```diff
@@ -78,6 +81,7 @@ namespace Aquamarine {
         };
 
         const SInternalState& state();
+        bool                  cursorCommited = false;
```

Comment:
is there any point for this to be publicly r/w?
can this just be a `bool cursorCommitted()`?

Reason is less potential ABI breaks